config.py

```diff
@@ -9,6 +9,7 @@ class Config(object):
     DEBUG = os.getenv('DEBUG', False)
     TESTING = False
     PORT = os.getenv('PORT', 8085)
+    GOOGLE_ANALYTICS = os.getenv('GOOGLE_ANALYTICS', None)
```

Comment:
Should this be `GOOGLE_ANALYTICS_ID` for clarity?

Reply:
Yes, but ras-frontstage uses this.  I figured consistency > being correct in this case. Or should I make this right to hopefully make future ones right?

Reply:
Semantically correct is always nice :)